### PR TITLE
Lokalise: Translations update

### PR DESCRIPTION
**Pull request by scireumskip**
This pull request was initiated by Lokalise (user Marcus Korinth) at 2024-11-27 09:41:26

### CODE REVIEW

File: src/main/resources/web_en.properties [hunk 1]
```diff
@@ -44,10 +44,14 @@ fileUpload.URL = URL
...
 help = Help
+selectableTable.all.deselection = Deselect all
+selectableTable.all.selection = Select all
+selectableTable.row.deselection = Deselect
+selectableTable.row.selection = Select
 table.html.empty = No contents found
 template.html.additionalActions = Additional Actions
 template.html.confirmHeader = Are you sure?
-template.html.confirmText = Press 'Yes' to continue or Cancel to go back.
+template.html.confirmText = Press 'Yes' to continue or 'Cancel' to go back.
```
Comment (jakobvogel) [inline src/main/resources/web_en.properties:54]: This line was only adapted in English, not in the other languages. What about the German version?

**Review by jakobvogel — APPROVED** [review]

**Review by scireum-mbo — APPROVED** [review]

**Review by idlira — APPROVED** [review]

File: src/main/resources/web_en.properties (hunk 1 above)
Comment (mko-sci) [reply src/main/resources/web_en.properties:54]: `Drücken Sie 'Ja', um fortzufahren oder 'Abbrechen', um zurückzugehen.`

Comment (mko-sci) [reply src/main/resources/web_en.properties:54]: The english version has been enhanced by the quotes. The German original already came with quotes (although not the correct ones... but we need to fix the quotation in sirius anyway as this is not the only key)

**Review by mkeckmkeck — APPROVED** [review]

File: src/main/resources/web_en.properties (hunk 1 above)
Comment (jakobvogel) [reply src/main/resources/web_en.properties:54]: Look at _all_ the other languages as well, plz. There, the new quotes seem to be missing.

Comment (mko-sci) [reply src/main/resources/web_en.properties:54]: Yes, they are missing there indeed.

**Pull request merged**